experimental/packages/opentelemetry-sdk-metrics-base-wip/src/MetricExporter.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export abstract class MetricExporter {
+    private _shutdown = false;
+
+    async shutdown(): Promise<void> {
+        try {
```

Comment:
`shutdown()` should only be called once per the spec. I think this should look like `MetricReader.shutdown()` below

```ts
        if (this._shutdown) {
            return;
        }

        this._shutdown = true;
        // ...
```

Reply:
we're not calling it more than once, but we're just protecting against misuse. i'm not sure what change you're suggesting.

Reply:
I was suggesting we should protect against multiple calls to shutdown and warn, but I guess it's not clear from the spec.

Reply:
I added it to the abstract class but i'm not convinced this class will end up being in the final sdk anyway.